### PR TITLE
Vaughn/com 134 add ingredients

### PR DESCRIPTION
**Pull request by gregoryvm**

### CODE REVIEW

File: Food.json
Comment (gapy04) [inline Food.json:1]: When doing etl, we generally want to avoid putting data files into the repo because data can change and we don't want others to see what data we are adding. Please remove this file from the repo and instead put it into a shared folder where only team members can access and download i.e. team OneDrive.

File: temp.json
Comment (gapy04) [inline temp.json:1]: Remove test.json file

File: cupboard_app/v2_urls.py
Comment (gapy04) [inline cupboard_app/v2_urls.py:1]: Remove changes from the urls.py

File: cupboard_app/views.py
Comment (gapy04) [inline cupboard_app/views.py:1]: Let's change the approach so that we aren't putting this in views because we don't want people to publicly access or add ingredients to the database and we don't want to have to runserver everytime.
Instead, create a separate script called `add_ingredients.py` in the `utils/db_helper` folder and put the code you wrote into there. This will be like a one-time run script.
To run this script, we would do `python manage.py shell < utils/db_helper/add_ingredients.py`

File: cupboard_app/views.py [hunk 1]
```diff
@@ -769,3 +772,19 @@ def list(self, request: Request) -> Response:
         all_measurements = get_all_measurements()
         serializer = MeasurementSerializer(all_measurements, many=True)
         return Response(serializer.data)
+
+
+class UploadDataAPIView(APIView):
+    permission_classes = [AllowAny]
+
+    def get(self, request: Request) -> Response:
+        with open('temp.json', 'r', encoding="utf8") as file:
```
Comment (gapy04) [inline cupboard_app/views.py]: For the file, we'll have it so that we have a folder structure like so
/src/food.json
This folder isn't part of the repository and devs can add any data files into this directory and run this script to add the ingredients.

**Review by gapy04 — CHANGES_REQUESTED** [review]
Several changes to remove the script from the views and instead make this a one-time run script

File: temp.json
Comment (gregoryvm) [reply temp.json:1]: Realized that right after I made the PR :( 
Already done 👍

File: Food.json
Comment (gregoryvm) [reply Food.json:1]: Added full zip with all json files including Food.json to OneDrive, removed Food.json from repo 👍

File: cupboard_app/views.py (hunk 1 above)
Comment (gregoryvm) [reply cupboard_app/views.py]: done 👍

File: cupboard_app/v2_urls.py
Comment (gregoryvm) [reply cupboard_app/v2_urls.py:1]: done 👍

File: Food.json
Comment (gregoryvm) [reply Food.json:1]: Also added Food.json itself, which is different than the version in the zip

File: cupboard_app/views.py
Comment (gregoryvm) [reply cupboard_app/views.py:1]: done 👍

File: cupboard_app/src/Food.json
Comment (gapy04) [inline cupboard_app/src/Food.json:1]: Remove the src folder entirely. The src folder should not be part of the commit.

File: cupboard_app/v2_urls.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
     MeasurementsViewSet,
     UserViewSet,
     UserListIngredientsViewSet,
-    UpdateUserListIngredientsViewSet
+    UpdateUserListIngredientsViewSet,
```
Comment (gapy04) [inline cupboard_app/v2_urls.py]: Remove the comma as this file is no longer required in this change.

File: utils/db_helper/add_ingredients.py
Comment (gapy04) [inline utils/db_helper/add_ingredients.py:1]: For strings in python. Use single apostrophe ' instead of " as the rest of the repo is using this. However if you have to do something like 'don\'t' then use double quotes instead i.e. "don't". For docstrings however we use """ to follow docstring standard
https://peps.python.org/pep-0008/#string-quotes

**Review by gapy04 — CHANGES_REQUESTED** [review]

File: cupboard_app/src/Food.json
Comment (gregoryvm) [reply cupboard_app/src/Food.json:1]: done 👍

File: cupboard_app/v2_urls.py (hunk 2 above)
Comment (gregoryvm) [reply cupboard_app/v2_urls.py]: done 👍

File: utils/db_helper/add_ingredients.py
Comment (gregoryvm) [reply utils/db_helper/add_ingredients.py:1]: done 👍

**Review by gapy04 — APPROVED** [review]
Lovely!
